src/pdf/ShippingLabels/ShippingLabelsPdf.tsx

```diff
@@ -148,7 +148,7 @@ interface ShippingLabelsForSingleParcelProps {
 const ShippingLabelsForSingleParcel: React.FC<ShippingLabelsForSingleParcelProps> = ({ data }) => {
     return (
         data.label_quantity > 0 &&
-        [...Array(data.label_quantity)].map((value: undefined, index: number) => {
+        [...Array(data.label_quantity)].map((index: number) => {
```

Comment:
the map accepts a function where the function parameters are
- item in the array
- index of the item
- the array itself
in that order.

If we only want to use the index, we must do `(_, index) => { ...`

[map docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)

```suggestion
        [...Array(data.label_quantity)].map((_, index) => {
```

Reply:
Yes got it thanks 👍 - linting doesn't like short identifiers - should I es-lint ignore?
![image](https://github.com/NorwoodAndBrixtonFoodbank/nbf-website/assets/160640328/c60c4e38-0092-4928-b698-3d411a16af25)